### DISCUSSION TITLE
Reducing CPU/memory/network bandwitdth attack attempts on IPsec

### DISCUSSION THREAD

**Discussion by metalefty** [SoftEtherVPN/SoftEtherVPN#1461] [Ideas]
Hi, 

as you might already know, stable version implemented some mitigations for attack attempts on IPsec.
I think this is also needed for developer version if it has the same resource consumption issue.

Changes are here: https://github.com/SoftEtherVPN/SoftEtherVPN_Stable/compare/v4.36-9754-beta...v4.38-9760-rtm

See also the entry at **SoftEther VPN 4.37 Build 9758 Beta** and **SoftEther VPN 4.38 Build 9760 RTM**.
https://www.softether.org/5-download/history

**Comment by chipitsine** [comment]
I beleive there were real justification for IPSec attack mitigations, but I did not heard of such attack patterns.
just for the record, are there any articles ?

**Reply by metalefty** [reply]
@dnobori wrote an article in Japanese.
https://ja.softether.org/9-about/news/901-210816
